### DISCUSSION TITLE
懶很好

### DISCUSSION THREAD

**Discussion by JessyCheng2003** [1101-GS4719-C/Python-Community-GS4719-C#1418] [General]
昨天晚上看到蔡康永的訪談影片 ，平常都是他當主持人訪問別人，這次是他被訪問，也讓我發現他是一個很有內涵和具有文學底蘊的人；在這支影片最讓我深刻的一句話是 「懶很好」，蔡康永覺得懶很好，懶能促使我們做出很有效率的事，像是人因為不想走路，就發明了輪子，這讓我想到好像就是因為「懶」而促使科技的進步，因為「懶」而使人類想出更有效率的方法或發明，所以生活中讓你覺得懶或是麻煩的事，其實都是一個契機，你可以選擇腳踏實地慢慢的做完，也可以選擇想出更棒的方式或是發明去解決它。聽完蔡康永的一些話，讓我了解到文字的力量，也讓我拾回對文學的興趣了。最後推薦讀理工科系的大家，平時可以接觸一點文學，它能讓你豁然開朗或是對事情有新的見解。

**Comment by TW-Maverick** [comment]
這邊的「懶」我覺得是屬於懶得做某件事，而發明出更快速或更方便的東西，跟「懶得動」只想耍廢字面上可能一樣，但本質上好像不太一樣，我自己認為我是後者！
像我現在越來越少有閱讀的習慣，之後我會試著去改變👌

**Comment by Jamesmoneya** [comment]
我真的很懶會不想做任何事只想耍廢，但是出於責任感，我會在最後的時間，用最有效率的方式把事情完成，這樣也是懶惰出新境界

**Comment by PaulSu0905** [comment]
我覺得蔡康永的觀點確實很有啟發性，他以一個獨特的角度看待懶惰，將其視為一種激勵創造力和效率的力量。這種思考方式也可以應用到生活中的各個方面。

**Comment by bestbanana295** [comment]
我覺得懶在某種程度上不會對自己造成太大的壓力，只要不要太誇張其實懶惰一點也無所謂。

**Comment by larry40305** [comment]
面對這些困難，我們可以選擇用傳統的方式去克服，但同時也可以思考更優化的方法或創造新的解決方案。這種思考過程可能會促使我們發現更高效、更創新的解決方案，同時提升我們的創造力和解決問題的能力。

**Comment by slps101023** [comment]
蔡康永的「懶很好」這個觀點確實令人耳目一新，將「懶」與創新效率巧妙地結合在一起，讓人重新審視日常的慣性思維。他的文學底蘊和思考方式不僅讓人感到啟發，也展現了文字能帶來的深刻力量。

**Comment by lai1008** [comment]
我覺得懶在某種意義上是因為不會對自己造成太大的壓力所以才會想要懶一點，而且只要不要太誇張該做的都有做完其實懶惰一點也無所謂。

**Comment by wcyeee** [comment]
我覺得這個想法很棒！沒想到被大家認為的負面的形容詞懶，也可以是一個激發創意的機會！我覺得這讓我以後如果遇到什麼不想做的事，不會再單純拖延等等的，可以試著往怎麼更快解決他的方向想，不只可以更快解決事情，也可以想出更棒的解決方法！

**Comment by yue083139** [comment]
我覺得懶惰的想法會促使我更有效率的解決我的工作，像我最近就很喜歡躺在床上一動不動休息，但每次要工作的時候我就會為了在冷冷的天躺到床裡面而提高效率，在床上的時候也可以重新思索等等的代辦事項及優先順序，我想懶惰也可以成為一種努力的原動力吧。
